### DISCUSSION TITLE
카메라 회전 속도와 줌 속도를 구분해서 조절하는 기능 문의

### DISCUSSION THREAD

**Discussion by mhkim0720** [naverz/zepeto-studio-kor#132] [5. Q&A]
안녕하세요 

구현 하다 보니 카메라 회전 속도와 줌 속도를 구분해서 조절이 필요한 부분이 있는데 
지원하는 기능일까요? 

![image](https://user-images.githubusercontent.com/8485731/147198114-65702ea5-7047-452e-b8a2-b445db3d274e.png)

위 이미지에 Speed / Sensitivity 값을 조정하면 카메라 회전 속도와 줌 속도가 같이 변하는 거 같습니다. 

확인 부탁 드립니다.

**Comment by moondory77-naverz** [comment]
안녕하세요.

 ZepetoPlayers 컴포넌트에서, Camera >> Sensitivity / Speed  값은
방식에는 차이가 있으나, 모두 Zoom / Rotation 민감도에 영향을 주는 수치입니다.

Sensitivity : input Event 반영값 (delta time)
Speed : output Event 반영값 (delta time) 

말씀주신대로, 현재 Camera의 (Zoom / Rotation) 동작에
Sensitivity, Speed 두개의 수치가 함께 적용되고있기 때문에
구분하여 조절하는 것은 어렵습니다.

마찬가지로 다음 업데이트를 통해, 
각 동작에 대해 구분하여 수치값을 적용할 수 있는 기능을 제공드릴 수 있도록 하겠습니다. 

감사합니다.
